### PR TITLE
Fix Delete /display names STL

### PR DESCRIPTION
**Pull request by RossStewart**

### CODE REVIEW

File: tests/Unit/Facade.Tests/StockTriggerLevelsServiceTests/WhenAdding.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    using Linn.Stores.Domain.LinnApps.StockLocators;
+    using Linn.Stores.Resources;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenAdding : ContextBase
+    {
+        private readonly int stockTriggerLevelId = 1;
```
Comment (lewisrenfrew) [inline tests/Unit/Facade.Tests/StockTriggerLevelsServiceTests/WhenAdding.cs:15]: var looks unused

File: tests/Unit/Facade.Tests/StockTriggerLevelsServiceTests/WhenAdding.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        private StockTriggerLevelsResource resource;
+
+        private IResult<StockTriggerLevel> result;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var stockTriggerLevel = new StockTriggerLevel
```
Comment (lewisrenfrew) [inline tests/Unit/Facade.Tests/StockTriggerLevelsServiceTests/WhenAdding.cs:24]: is this used?

File: tests/Unit/Facade.Tests/StockTriggerLevelsServiceTests/WhenAdding.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                TriggerLevel = 400
+            };
+
+            this.result = this.Sut.Add(this.resource);
+        }
+
+        [Test]
+        public void ShouldAdd()
+        {
+            this.Repository.Received().Add(Arg.Any<StockTriggerLevel>());
```
Comment (lewisrenfrew) [inline tests/Unit/Facade.Tests/StockTriggerLevelsServiceTests/WhenAdding.cs:52]: What are we actually testing here? the fact that the repository is called and that the Created varaint of the IResult are returned are just functions of the common library code - you shouldn't be unit testing them here. If you want to test the /create route an integration test would be a better place to do so

Comment (RossStewart) [reply tests/Unit/Facade.Tests/StockTriggerLevelsServiceTests/WhenAdding.cs:52]: Fair point. It's a fairly standard Add so I suppose I could take this out.

**Review by lewisrenfrew — APPROVED** [review]

**Pull request merged**